pages/api/popular/api_popular.js

```diff
@@ -0,0 +1,9 @@
+import fetcher from "@/services/fetch";
+
+export default async function handler(request, response) {
+  // const { page } = request.query;
+  const { page } = request.query;
+  const url = `https://api.themoviedb.org/3/movie/popular?api_key=${process.env.API_KEY}&language=en-US&page=${page}`;
+  const movieData = await fetcher(url);
+  response.status(200).json(movieData);
```

Comment:
What happens if there is an error?

Reply:
The retrieval error is handled in the PopularList.js, UpcomingList.js and TopRatedList.js components